.github/workflows/pypi.yml

```diff
@@ -1,6 +1,8 @@
 name: Upload Python Package
 
 on:
+  push:
+    branches: [main, dev]
```

Comment:
do you still want to keep this as the trigger for running PyPi? or should it be back to only running on a new GitHub release?